buildSrc/src/main/kotlin/publishing-conventions.gradle.kts

```diff
@@ -23,6 +23,7 @@ tasks.withType<PublishToMavenLocal>().configureEach {
 }
 
 signing {
+	useGpgCmd()
```

Comment:
❓ Why did you add this?

Reply:
In order to test I had to `releaseToMavenLocal` which triggers signing. The signing plugin has a long standing bug on Linux. The official fix is to apply the above line. As referenced in the commit message see https://docs.gradle.org/current/userguide/signing_plugin.html#sec:using_gpg_agent. Probably better to just drop the commit from this PR. Stand by...